### PR TITLE
Create layer app final

### PR DESCRIPTION
**Pull request by capooti**
Backporting the createlayer app to 2.7.x

### CODE REVIEW

**Comment by capooti** [comment]
Oh, I see there are also the other commits from master: should I recreate a PR without them?

File: geonode/__init__.py [hunk 1]
```diff
@@ -20,7 +20,7 @@
 
 import os
 
-__version__ = (2, 7, 0, 'alpha', 0)
+__version__ = (2, 9, 0, 'unstable', 0)
```
Comment (simod) [inline geonode/__init__.py]: Paolo please unstage this one

File: pavement.py [hunk 2]
```diff
@@ -661,7 +661,7 @@ def versions():
     if stage == 'final':
         stage = 'thefinal'
 
-    if stage == 'alpha' and edition == 0:
+    if stage == 'unstable':
```
Comment (simod) [inline pavement.py]: this as well

File: geonode/version.py [hunk 3]
```diff
@@ -29,7 +29,7 @@ def get_version(version=None):
         from geonode import __version__ as version
     else:
         assert len(version) == 5
-        assert version[3] in ('alpha', 'beta', 'rc', 'final')
```
Comment (simod) [inline geonode/version.py]: this as well

**Pull request closed without merging**